docs/schemas/itemMeta.json

```diff
@@ -0,0 +1,996 @@
+          "enum": ["singleLine"]
+        },
+        "minSize": {
+          "description": "Min number of characters",
+          "type": "number",
+          "default": 1
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 140
+        }
+      }
+    },
+
+    "tas-multiline": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-multiline",
+      "title": "tas-multiline",
+      "description": "A multiline text question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["multiline"]
+        },
+        "minSize": {
+          "description": "Min number of characters",
+          "type": "number",
+          "default": 1
+        },
+        "maxSize": {
+          "description": "Max number of characters",
+          "type": "number",
+          "default": 1000
+        }
+      }
+    },
+
+    "tas-number": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-number",
+      "title": "tas-number",
+      "description": "A number question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["number"]
+        },
+        "maximum": {
+          "description": "Max number size",
+          "type": "number"
+        },
+        "minimum": {
+          "description": "Min number size",
+          "type": "number"
+        }
+      }
+    },
+
+    "tas-bool": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-bool",
+      "title": "tas-bool",
+      "description": "A boolean question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["bool"]
+        },
+        "trueString": {
+          "description": "String that represents a true answer",
+          "type": "string",
+          "default": "Yes"
+        },
+        "falseString": {
+          "description": "String that represents a false answer",
+          "type": "string",
+          "default": "No"
+        }
+      }
+    },
+
+    "tas-resume": {
+      "$schema": "http://json-schema.org/draft-04/schema",
+      "id": "http://talentappstore.github.io/tas-tenant-apis/tas-resume",
+      "title": "tas-resume",
+      "description": "A resume question item",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["resume"]
+        },
+        "version": {
+          "description": "Previous version ID",
+          "type": "string"
+        },
+        "job": {
+          "description": "ID for the job associated with this item",
+          "type": "string"
+        },
+        "candidate": {
```

Comment:
I don't see why candidate ID ever needs to be in here, you are always within the scope of a candidate (or application) when referring to a resume.